tests/posix/common/src/signal.c

```diff
@@ -5,12 +5,16 @@
  */
 
 #include <errno.h>
+#include <pthread.h>
 #include <signal.h>
 #include <stdio.h>
 
-#include <zephyr/kernel.h>
+#include <zephyr/sys/util.h>
 #include <zephyr/ztest.h>
 
+/* A 32-bit value to use between threads for validation */
+#define BIOS_FOOD 0xB105F00D
```

Comment:
A more descriptive name might be good here? What is this value used for?

Reply:
fixed!